SAMv1.tex

```diff
@@ -192,6 +192,12 @@ \subsection{The header section}
     \emph{Valid values}: {\tt none} (default), {\tt query} (alignments are
     grouped by {\sf QNAME}), and {\tt reference} (alignments are grouped by
     {\sf RNAME}/{\sf POS}).\\\cline{1-3}
+  & {\tt SS} & Sub-Sorting order of alignments. Valid values are of the form
+    {\tt <sort-order>:<sub-sort>}, where {\tt <sort-order>} is the same value stored in the
+    {\tt SO} attribute and {\tt <sub-sort>} is an implementation-dependent string further 
```

Comment:
can you add a regex limiting the scope of the subsort string?

Reply:
Or possibly a combination of regex and textual description, eg colon separated list of sort orders, each of which must match the regex `[A-Za-z0-9_]+`? (ie alphanumeric plus underscore).